src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -183,7 +206,11 @@ export default function OpeningPeriodForm({
               labelText="Alkaa"
               onChange={(value): void => setPeriodBeginDate(value || null)}
               value={periodBeginDate}
+              error={errors.openingPeriodBeginDate}
               registerFn={register}
+              customValidations={{
```

Comment:
I wonder would it be better to have the validation in the end time input field, as the end time is often inserted last, and it is the field where user was/is when the error is prompted?

Reply:
Yes. It makes sense, I'll fix